### DISCUSSION TITLE
I search for  year/month selection component

### DISCUSSION THREAD

**Discussion by sergeynilov** [livewire/livewire#8521] [1. Help] [answered]
I search for livewire 3 some year/month selection input component. Multilanguage, please. 

Are there any ?

**Comment by Akhmami** [comment]
I use flatpickr, you can see it [here](https://github.com/Akhmami/anonymous-blade-component/blob/main/views/components/input/date-month.blade.php)
for multilanguage see the flatpickr documentation

**Comment by sergeynilov** [comment]
Thank you!

In resources/js/bootstrap.js file I added :

```
import monthSelectPlugin from 'flatpickr/dist/plugins/monthSelect';
window.monthSelectPlugin = monthSelectPlugin;

```

and adding code

```
    <div wire:ignore class="px-2 w-full">
        <x-inputs.date-month id="year_month" wire:model="filters.year_month" />
    </div>

```
I see date time component

https://img001.prntscr.com/file/img001/sUPVG9bJS_qoYvgDqzJFZg.png

But broken - looks like relative css classes are missing... Where can I get these css classes?

**Reply by Akhmami** [reply, marked as answer]
add flatpicker style to your css file
```css
@import "flatpickr/dist/flatpickr.min.css";
@import "flatpickr/dist/plugins/monthSelect/style.css";
```
